libs/labelbox/src/labelbox/schema/export_task.py

```diff
@@ -106,7 +108,14 @@ class JsonConverterOutput:
 
 
 class JsonConverter(Converter[JsonConverterOutput]):  # pylint: disable=too-few-public-methods
-    """Converts JSON data."""
+    """Converts JSON data.
+    
+    Deprecated: This converter is deprecated and will be removed in a future release.
+    """
```

Comment:
The _find_json_object_offsets method leads to coding errors (implemented incorrectly), trying to parse chunks for json will lead to errors, buffer is guaranteed to be accurate